projects/ngx-formentry/src/components/radio-button/radio.component.html

```diff
@@ -0,0 +1,9 @@
+<fieldset class="bx--fieldset">
```

Comment:
Late to this @icrc-toliveira, just a note about the carbon design prefix, should we be using `bx--` or `cds--`?